storage/innobase/include/log0recv.h

```diff
@@ -52,6 +52,14 @@ class PersistentTableMetadata;
 
 struct recv_addr_t;
 
+#ifdef XTRABACKUP
+/** map of tablespace_id, that experienced an inplace DDL during a backup op */
+extern std::map<space_id_t, bool> index_load_map;
+extern bool opt_page_tracking;
+extern char *xtrabackup_incremental;
+extern lsn_t incremental_start_checkpoint_lsn;
+#endif /* XTRABACKUP */
+
```

Comment:
those all comes from `xtrabackup.cc` their logical place is where they are declared and not where they are used.
Xtrabackup uses xtrabackup.h as header file for xtrabackup related source code( the ones under storage/innobase/xtrabackup/src`) and xb0xb.h for shared code between xtrabackup and server/innodb. 
Those extern's belong to xb0xb.h

Reply:
fixed